### ISSUE TITLE
Clicking on  Contributor's Guide's link directs to https://docs.dev.to/contributing/forem/

### ISSUE THREAD

**Issue by cmgorton** [forem/forem#9505]
**Describe the bug**

Clicking on  Contributor's Guide's link directs to https://docs.dev.to/contributing/forem/ instead of https://docs.forem.com/

**To Reproduce**

1. Go to https://github.com/forem/forem/blob/master/CONTRIBUTING.md
2. Click on contributor's guide link
3. You will be directed to https://docs.dev.to/contributing/forem/

**Expected behavior**

Should link to Forem's docs instead https://docs.forem.com/

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @forem/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/forem/forem/blob/master/CONTRIBUTING.md).

**Comment by cmgorton** [comment]
If it is up for grabs I would be happy to work on the issue.

**Comment by cmgorton** [comment]
I found another area with the same issue. I could put in another issue for it if you think that is necessary. On this link:
https://github.com/forem/forem#getting-started

When you click the Developer Documentation link under getting started it also directs you to  https://docs.dev.to/ instead of the newer forem docs link.

**Comment by rhymes** [comment]
@cmgorton I think you can fix both in the same issue if you'd like! Assigned this to you!
